### ISSUE TITLE
why NiftiMasker message is "Loading data from None" ?

### ISSUE THREAD

**Issue by salma1601** [nilearn/nilearn#1197]
just wondering

**Comment by AlexandreAbraham** [comment]
This is not supposed to happen, I think.

**Comment by salma1601** [comment]
For me it happens when re-running this code
```Python
from nilearn import datasets, input_data
adhd = datasets.fetch_adhd(n_subjects=1)
icbm = datasets.fetch_icbm152_brain_gm_mask()
masker = input_data.NiftiMasker(mask_img=icbm, memory='nilearn_cache',
                                verbose=1)
signals = masker.fit_transform(adhd.func[0])
```
prints
>[NiftiMasker.fit] Loading data from None
[NiftiMasker.fit] Resampling mask

but when I do not define the `mask_img`, the message is OK.

**Comment by KamalakerDadi** [comment]
Why not we do ```fit(imgs, y=None)``` rather than ```fit(imgs=None, y=None)``` in NiftiMasker and MultiNiftiMasker ?

**Comment by bthirion** [comment]
Stalled. Closing.
